src/igvfd/schemas/in_vitro_system.json

```diff
@@ -590,6 +590,19 @@
                 "appscript": "DMEM with serum",
                 "igvf_utils": "DMEM with serum"
             }
+        },
+        "biosample_qualifier": {
+            "title": "Biosample qualifier",
+            "description": "Desription of cell state",
+            "type": "string",
```

Comment:
Should it be an array? Wondering if it is possible to have 2 different qualifiers? Seem to recall Ben mentioning it but not sure if we decided.